test/configCases/asset-modules/url-prefetch-preload-fetchpriority/index.js

```diff
@@ -0,0 +1,170 @@
+"use strict";
+
+// Warnings are generated in generate-warnings.js to avoid duplication
+
+// Mock for document.head structure
+global.document = {
+	head: {
+		_children: [],
+		appendChild: function(element) {
+			this._children.push(element);
+		}
+	},
+	createElement: function(tagName) {
+		const element = {
+			_type: tagName,
+			_attributes: {},
+			setAttribute: function(name, value) {
+				this._attributes[name] = value;
+				// Also set as property for fetchPriority
+				if (name === 'fetchpriority') {
+					this.fetchPriority = value;
+				}
+			},
+			getAttribute: function(name) {
+				return this._attributes[name];
+			}
+		};
+		return element;
+	}
+};
```

Comment:
We have `moduleScope` function to inject something in global scope, or feel free to improve our DOM mock for tests, you can use another PR for this

Reply:
fixed in c632fcc9eef369f2b1f7deddefc464427a748e4b